website/docs/r/container_nlb_dns.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+subcategory: "Kubernetes Service"
+layout: "ibm"
+page_title: "IBM : ibm_container_nlb_dns"
+description: |-
+  Manages IBM Container Nlb
+---
+
+# ibm_container_nlb_dns
+
+Provides a resource for container_nlb_dns. This allows container_nlb_dns to be created, updated and deleted.
```

Comment:
This resource will only support only Add
 add - Add an NLB IP to an existing host name that you created with 'ibmcloud ks nlb-dns create'.

Reply:
made changes to doc